R/matching.R

```diff
@@ -0,0 +1,140 @@
+#' Relaxed Value Matching
+#'
+#' These functions offer relaxed matching of one vector in another.
+#' In contrast to the similar [`match`] and [`%in%`] functions they
+#' just accept `numeric` arguments but have an additional `tolerance`
+#' argument that allows relaxed matching.
+#'
+#' @param x `numeric`, the values to be matched.
+#' @param table `numeric`, the values to be matched against. In contrast to
+#' [`match`] `table` has to be sorted in increasing order.
+#' @param tolerance `numeric`, accepted tolerance. Use `0` to get the
+#' same behaviour as [`match`]. Use `Inf` (default) to match to the closest
+#' value without restrictions. Could be of length one or the same length as
+#' `table`.
+#' @param duplicates `character(1)`, it is possible that many `x` match one
```

Comment:
Just a minor thing: I find it easier for the user if parameter descriptions are short eventually referring to `@description` or `@note`. Suggestion: move this lengthy description to `@note` and shorten the parameter description.